packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java

```diff
@@ -18,6 +18,8 @@
  */
 
 import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
```

Comment:
https://github.com/invertase/react-native-firebase/blob/master/packages/app/android/src/main/java/io/invertase/firebase/common/UniversalFirebasePreferences.java#L26

We have a util for shared preferences which we use in other modules, maybe best using this?

Reply:
OK. Let me try using this module. Thank for noticing me that.

Reply:
@kyunkakata I'm interested in collaborating with you to get this merged, but it looks like there is a review comment here that still needs to be addressed

Sorry for the delay in checking in on this!